.markdown-link-check.json

```diff
@@ -1,6 +1,8 @@
 {
   "retryOn429": true,
-  "retryCount": 3,
+  "retryCount": 5,
+  "fallbackRetryDelay": "30s",
```

Comment:
From https://github.com/tcort/markdown-link-check#config-file-format

>- `fallbackRetryDelay` the delay in [zeit/ms](https://www.npmjs.com/package/ms) format. (e.g. `"2000ms"`, `20s`, `1m`) for retries on a 429 response when no retry-after header is returned or when it has an invalid value. Default is `60s`.

So setting this to `30s` is actually a tightening rather than a relaxation as described. Is that intentional?

Reply:
Ah, yes, the config file comes from the example [here](https://www.npmjs.com/package/markdown-link-check#:~:text=Zm9vOmJhcg%3D%3D%22%2C%0A%20%20%20%20%20%20%20%20%22Foo%22%3A%20%22Bar%22%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%5D%2C-,%22timeout%22%3A%20%2220s%22%2C,-%22retryOn429%22%3A%20true%2C%0A%20%20%22retryCount) as suggested by @rsora, I just checked the `Timeout` value thas is actually bigger than the default (10s) so I didn't bother the check the other.

Yes I think we could remove the `fallbackRetryDelay` from the config, I'll try that.